Symfony/CS/Tests/Fixer/Symfony/ArrayElementNoSpaceBeforeCommaFixerTest.php

```diff
@@ -78,26 +78,26 @@ public function testFixProvider()
             // multi line array
             array(
                 '<?php $x = [  "a" =>$a,
-                    "b"=>  
+                    "b"=>
```

Comment:
This cases tests completely different thing, I can't see any benefits of that spaces.